### DISCUSSION TITLE
send email automatically based on condition

### DISCUSSION THREAD

**Discussion by meilensteindigital** [liedekef/events-made-easy#119] [General]
i have a custom field which represents the minimum number of participants of an event. also i have set the event to stop letting people book the event 30 days before it starts. if the number of bookings is less than the minimum number of participants at the point of 30 days before the event starts, then the event will be canceled. when this happens, i want to send an email to the contact person automatically to inform that the event needs to be canceled. 

how can i do this? .. i guess it requires to insert an add_action to the functions.php, but as i'm not a big programmer i need some help at this point.

thank you 
best Thomas

**Comment by liedekef** [comment]
functions.php won't help you for something you want to check on a regular basis.
You'll need to hire someone for this: sql to check on the eventids that will close in 30 days, then use the eme functions to get the event, the number of seats etc ...

**Comment by meilensteindigital** [comment]
Hi Franky,

could you do this as a paid job? 
If so, Please me know how much it would propably cost. 
i will ask then my Client how important this function is for him and if he is willing to pay an extra for it.


Viele Grüße 
Thomas Jahn


Wir sind 3D - 3D Visualisierungen - 3D Animationen - Virtual Reality - Virtual Video Production - Virtuelle Messen
Meilenstein Digital GmbH
Thomas Jahn / Geschäftsführer
Oskar-von-Miller-Str. 19
86199 Augsburg
Tel.: +49 821 5439355 1
Fax: +49 821 5439355 5
Mobil: +49 179 2110337
***@***.***
Web: https://www.meilenstein-digital.de
  

Von: Franky Van Liedekerke
Gesendet: Freitag, 24. März 2023 16:41
An: liedekef/events-made-easy
Cc: meilensteindigital; Author
Betreff: Re: [liedekef/events-made-easy] send email automatically based oncondition (Discussion #119)

functions.php won't help you for something you want to check on a regular basis.
You'll need to hire someone for this: sql to check on the eventids that will close in 30 days, then use the eme functions to get the event, the number of seats etc ...

**Reply by liedekef** [reply]
Please mail me at: franky [at] e-dynamics.be for a quote on this.
